src/Card/Card.scss

```diff
@@ -1,2 +1,140 @@
 @import "variables";
 @import "~bootstrap/scss/card";
+
+/**
+ * Card
+ */
+
+ .pgn__card {
+  .full-card-click:after {
+    // Any element using this class will become the element that receives a click
+    // when any part of the card is clicked.
+    //
+    // Note: any element that should be "lifted" outside of this click should be relatively
+    // positioned, e.g. action buttons in the footer should remain clickable outside of the
+    // full card click.
+    
+    // TODO: make this class be added conditionally on some component via a prop rather than explicitly
+    // passing ``className="full-card-click"``.
+    content: '';
+    position: absolute;
+    left: 0;
+    top: 0;
+    right: 0;
+    bottom: 0;
+  }
+ }
+
+/**
+ * Card Header
+ */
+
+ .pgn__card-header {
+  @extend .card-body;
+  @extend .pb-0;
```

Comment:
I prefer we use the SASS variables instead of extending css utils. We have $spacer-0 through 5 I think

Reply:
The sass utils will sprinkle `!important` in our styles

Reply:
Ah, yes. I noticed that while putting this together! Good idea to use variables here instead 👍